### PR TITLE
Pass EXPORTED_FUNCTIONS to lld via --export so it doesn't GC them

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: emscripten.py [hunk 1]
```diff
@@ -1895,8 +1894,15 @@ def debug_copy(src, dst):
       '--entry=main',
       '--allow-undefined',
       '--import-memory',
-      '--export', '__wasm_call_ctors',
-    ])
+      '--export', '__wasm_call_ctors']
+    for export in shared.expand_response(settings['EXPORTED_FUNCTIONS']):
+      cmd.append('--export')
```
Comment (jgravelle-google) [inline emscripten.py]: Nit: slightly terser as `cmd += ['--export', export[1:]]`

File: emscripten.py [hunk 2]
```diff
@@ -1895,8 +1894,15 @@ def debug_copy(src, dst):
...
+      cmd.append(export[1:]) # Strip the leading underscore
+    shared.check_call(cmd)
+
+    if DEBUG:
+      logging.debug('  emscript: lld took %s seconds' % (time.time() - t))
```
Comment (jgravelle-google) [inline emscripten.py]: Why did this move?

Comment (sbc100) [reply emscripten.py]: So it reports the actually time that lld took, and not the other stuff we are doing here.

Comment (jgravelle-google) [reply emscripten.py]: Ah right, we split this out from the wasm backend log earlier. I got confused at some point and thought it was timing all of `build_wasm_lld`

**Pull request merged**